src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,60 @@
+package duke;
+
+import duke.operation.TaskList;
+import duke.storage.Storage;
+import duke.ui.Ui;
+import duke.Parser.Parser;
+import duke.exception.DukeException;
+import duke.operation.Command;
+
+import java.util.Scanner;
+
+/**
+ * Duke is the smart assistance to help you track personal tasks.
+ * This is the Driver class for Duke application.
+ */
+public class Duke {
+
+	public static void main(String[] args) {
+		Ui.showWelcomeMessage();
+
+		// ask for user input
+		Scanner in = new Scanner(System.in);
+		String nextLine = in.nextLine();
+		Command nextCommand = Command.getCommandWordFromString(nextLine);
+		Storage storage = new Storage();
+		storage.createFile();
+		TaskList taskList = new TaskList();
+
+		while (true) {
```

Comment:
Maybe you can separate out this logic into a factory class?

Reply:
Yes I can. Maybe I'll do it some time later.